src/supercontrast/utils/text.py

```diff
@@ -1,3 +1,53 @@
+import re
+import unicodedata
+
+from num2words import num2words
+
+
 def truncate_text(text: str, max_bytes: int = 5000):
     """Truncate text to a maximum of max_bytes bytes."""
     return text.encode("utf-8")[:max_bytes].decode("utf-8", "ignore")
+
+
+def normalize_text(text: str, task: str = "transcription") -> str:
```

Comment:
this looks like the kind of function we should have some unit tests for

Reply:
that's definitely true, will add a basic test and flesh it out in a future PR